README.md

```diff
@@ -46,38 +33,68 @@ As a customer
+|            | @order = Order.new        | add_to_order(dish)    |
+| Order      | @order number             | view_basket           |
+|            | @basket = []              | calculate_total       |
+|            | @total = 0                | total_correct?        |
+|            | @history = []             | complete_order        |
+|            | @text = Text.new          | add_to_basket(dish)   |
+|            |                           | reset_basket          |
+|            |                           | reset_total           |
+|            |                           | reset_order           |
+|            |                           | order_history         |
+| Text       | @client (using twilio)    | send_text             |
+
+
+Optional extras:  
+add_to_menu  
+take_off_menu  
+dish_not_available  
+
+additional notes:  
+* menu is currently hardcoded to one menu list that initializes with every new restaurant instance  
+
 
-* Hints on functionality to implement:
-  * Ensure you have a list of dishes with prices
-  * The text should state that the order was placed successfully and that it will be delivered 1 hour from now, e.g. "Thank you! Your order was placed and will be delivered before 18:52".
-  * The text sending functionality should be implemented using Twilio API. You'll need to register for it. It’s free.
-  * Use the twilio-ruby gem to access the API
-  * Use the Gemfile to manage your gems
-  * Make sure that your Takeaway is thoroughly tested and that you use mocks and/or stubs, as necessary to not to send texts when your tests are run
-  * However, if your Takeaway is loaded into IRB and the order is placed, the text should actually be sent
-  * Note that you can only send texts in the same country as you have your account. I.e. if you have a UK account you can only send to UK numbers.
+#### Things that could be added:  
 
-* Advanced! (have a go if you're feeling adventurous):
-  * Implement the ability to place orders via text message.
+[ ] ensure an order can't be completed if nothing in the basket  
+[ ] minimum total of £15 for each order  
+[ ] A customer can delete from the order  
 
-* A free account on Twilio will only allow you to send texts to "verified" numbers. Use your mobile phone number, don't worry about the customer's mobile phone.
+----
+#### IRB testing
 
-> :warning: **WARNING:** think twice before you push your **mobile number** or **Twilio API Key** to a public space like GitHub :eyes:
->
-> :key: Now is a great time to think about security and how you can keep your private information secret. You might want to explore environment variables.
+require './lib/restaurant.rb'
 
-* Finally submit a pull request before Monday at 9am with your solution or partial solution.  However much or little amount of code you wrote please please please submit a pull request before Monday at 9am
+##### create a restaurant instance:  
+sparkleBurgers = Restaurant.new("Sparkle Burgers LTD")
```

Comment:
You added a full name of the restaurant as a string which  will be a very useful feature